pyo3-macros-backend/src/pyclass.rs

```diff
@@ -407,6 +407,57 @@ pub fn build_py_enum(
     Ok(impl_enum(enum_, &args, doc, method_type))
 }
 
+fn get_enum_variant_python_name<'a>(
```

Comment:
Could probably be a method `PyClassEnumVariant::python_name` if options were stored in the struct as per above.